pinot-core/src/main/java/com/linkedin/pinot/core/realtime/impl/kafka/CombinedStreamProviderConfig.java

```diff
@@ -36,7 +35,7 @@
 import static com.linkedin.pinot.common.utils.EqualityUtils.isSameReference;
 
 
-public class KafkaHighLevelStreamProviderConfig implements StreamProviderConfig {
+public class CombinedStreamProviderConfig implements StreamProviderConfig {
```

Comment:
This is a kafka specific class, right? Why call it a generic name?

Reply:
Reverted the renaming